lib/kitchen/driver/ec2.rb

```diff
@@ -159,6 +159,8 @@ def finalize_config!(instance)
 
         if config[:availability_zone].nil?
           config[:availability_zone] = config[:region] + "b"
+        elsif config[:availability_zone] =~ /^[a-zA-Z]$/
```

Comment:
Not at all, it just brings the behavior in line with what was already in the README